Storage/SQL/SQLiteHistory.swift

```diff
@@ -199,19 +237,22 @@ extension SQLiteHistory: BrowserHistory {
     }
 
     public func getSitesByFrecencyWithLimit(limit: Int) -> Deferred<Result<Cursor<Site>>> {
-        return self.getSitesByFrecency(limit: limit, filter: nil)
-    }
+        let groupBy = "GROUP BY \(TableHistory).domain_id "
```

Comment:
Ah yes, we're selecting places with visitCount and lastVisitDate and then grouping with an aggregate, rather than grouping on a subquery with an aggregate over the computed frecency. Tricksy.